apps/deck-playground/layers/vessel/VesselsLayer.ts

```diff
@@ -4,7 +4,7 @@ import { VesselLayer, VesselLayerProps } from 'layers/vessel/VesselLayer'
 export type VesselsLayerProps = { ids: string[] } & VesselLayerProps
 
 export class VesselsLayer extends CompositeLayer<VesselsLayerProps> {
-  vesselLayers = this.props.ids.map(
+  layers = this.props.ids.map(
```

Comment:
As this needs to be a convention, where do you think we could add the documentation?